doxia-core/src/test/java/org/apache/maven/doxia/sink/impl/AbstractSinkTest.java

```diff
@@ -115,9 +115,9 @@ public final void testTestDocument() throws IOException {
     }
 
     /**
-     * Checks that the sequence <code>[title(), text( title ), title_()]</code>,
+     * Checks that the sequence <code>[title(), text(title), title_()]</code>,
```

Comment:
these javadoc formatting updates add noise to the change: can we split them in a different PR (no need for any Jira issue...)

Reply:
Had the same concern. Will move them out today.

Reply:
I just did it and rebased your branch